pkg/curatedpackages/reader.go

```diff
@@ -143,7 +145,12 @@ func removeDuplicateImages(images []releasev1.Image) []releasev1.Image {
 	return list
 }
 
-func getRegistry(uri string) string {
+func getChartRegistry(_ string) string {
+	// For now, we only support prod as a source
+	return publicProdECR
```

Comment:
This wouldn't work for development then since the images might not be available in prod. Any reason we can't use dev registry?

Reply:
We could also have this on a follow up PR if that will be easier.

Reply:
Yeh, this could be better, but not the highest priority atm.

Reply:
https://github.com/aws/eks-anywhere/issues/4620

Reply:
Fixed.
Closes: https://github.com/aws/eks-anywhere/issues/4620